lib/embulk/command/embulk_migrate_plugin.rb

```diff
@@ -43,8 +43,8 @@ def self.migrate_java_plugin(migrator, from_ver)
     end
 
     # upgrade gradle version
-    if migrator.match("gradle/wrapper/gradle-wrapper.properties", /gradle-2\.\d-/)
-      # gradle < 2.10 (\d matches one digit)
+    if migrator.match("gradle/wrapper/gradle-wrapper.properties", /gradle-2\.\d+-/)
```

Comment:
Yea, you're right. We will need it. (I didn't notice what you were saying before. Sorry about it.)